package.json

```diff
@@ -44,29 +44,29 @@
   "devDependencies": {
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.2.2",
-    "@babel/plugin-proposal-class-properties": "^7.3.0",
-    "@babel/preset-env": "^7.3.1",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
-    "@material-ui/core": "^3.9.0",
+    "@material-ui/core": "^4.2.0",
     "@storybook/addon-actions": "^5.1.9",
     "@storybook/react": "^5.1.9",
     "babel-eslint": "^10.0.2",
     "babel-loader": "^8.0.6",
     "coveralls": "^3.0.2",
-    "enzyme": "^3.2.0",
-    "enzyme-adapter-react-16": "^1.6.0",
-    "enzyme-to-json": "^3.2.2",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "enzyme-to-json": "^3.3.5",
     "gh-pages": "^1.2.0",
-    "jest": "^24.1.0",
-    "react": "^16.5.2",
-    "react-dom": "^16.5.2",
-    "react-test-renderer": "^16.2.0",
+    "jest": "^24.8.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-test-renderer": "^16.8.6",
     "standard": "^13.1.0"
   },
   "peerDependencies": {
-    "@material-ui/core": "^1.0.0 || ^3.1.0",
-    "react": "^16.3.0",
-    "react-dom": "^16.3.0"
+    "@material-ui/core": "^1.0.0 || ^3.1.0 || ^4.0.0",
```

Comment:
It would be sufficient to only support `^4.0.0` here. People can use v1.0.0 of this component for compatibility with Material-UI 3.x